src/ui/settings/ViewAdvancedDNSSettings.qml

```diff
@@ -16,229 +16,100 @@ import telemetry 0.15
 
 
 Item {
+    property string settingId_default: "default"
```

Comment:
Why do we need this? Can we simply use the Q_ENUM entries?

Reply:
We can't do this, at least if i tried qml complained that we cant use script values as properties of a listmodel.

Reply:
Update: we __could__ if we are okay with creating the list model dynamicly

Reply:
We can get around this by sending only the string IDs in the ListModel and then `text: VPNl18n.tr(VPNl18n[tabLabelStringId])` to the button delegate in `VPNTabNavigation { }`. But this is working too!